uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/models/instances/leave-request-instance.js

```diff
@@ -92,7 +97,11 @@ define([
         defaultCustomData: function () {
           return {
             comments: [],
-            request_type: 'leave'
+            request_type: 'leave',
+            uploader: FileUpload.uploader({
+              entityTable: 'civicrm_hrleaveandabsences_leave_request',
+              crmAttachmentToken: sharedSettings.attachmentToken
+            })
```

Comment:
In the ticket you wrote

> files:[] is not required because the list of files to be uploaded is maintained by the 3rd party plugin FileUploader

but I'm not sure that's true. the 3rd party plugin is responsible for uploading the files, but when you load a leave request which already has files linked to it, how do you retrieve them? and where do you store them? that's what `files` was for. What's your alternative approach?